packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -60,6 +66,68 @@ export const defaultServices = {
 
   // Validation hooks for overriding
   async validateCustomSignUp(formData, touchData): Promise<ValidatorResult> {},
+  async validateFormPassword<Validator>(
+    formData,
+    touchData,
+    passwordSettings: PasswordSettings
+  ): Promise<ValidationComplexityError> {
+    const { password } = formData;
+
+    const { password: touched_password } = touchData;
+
+    /**
+     * If the password is not touched,
+     * or if the password settings are not set, we don't need to validate it.
+     */
+    if (!touched_password || !passwordSettings) return null;
+
+    const password_complexity = [];
+
+    const policyMinLength = +passwordSettings?.passwordPolicyMinLength;
+    if (password.length < policyMinLength) {
```

Comment:
What do you think of adding a `MINIMUM_LENGTH` rule? Would allow us to align this test with others in the below switch statement

Reply:
Will update!